mod/wikirate_source/spec/set/type_plus_right/source/file-import_spec.rb

```diff
@@ -2,191 +2,185 @@
-        expect(amazon_2015_metric_value_card.content).to eq("13")
-        expect(apple_2015_metric_value_card.content).to eq("9")
-        expect(sony_2015_metric_value_card.content).to eq("62")
+    context 'company correction name is filled' do
+      it 'should use the correction name as company names' do
+        Card::Env.params[:metric_values] = {"Amazon.com, Inc."=>["9"], "Apple Inc."=>["62"], "Sony Corporation"=>['13']}
+        Card::Env.params[:corrected_company_name] = {"Amazon.com, Inc."=>"Apple Inc.","Apple Inc."=>"Sony Corporation",  "Sony Corporation"=>'Amazon.com, Inc.'}
+        source_file = @source.fetch trait: :file
+        source_file.update_attributes subcards: {"#{@source.name}+#{Card[:metric].name}"=>{content: '[[Access to Nutrition Index+Marketing Score]]',type_id: Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{content: '[[2015]]',type_id: Card::PointerID}}
+
+        expect(Card.exists?('Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015')).to be true
+        expect(Card.exists?('Access to Nutrition Index+Marketing Score+Apple Inc.+2015')).to be true
+        expect(Card.exists?('Access to Nutrition Index+Marketing Score+Sony Corporation+2015')).to be true
+
+        amazon_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+value']
+        apple_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Apple Inc.+2015+value']
+        sony_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Sony Corporation+2015+value']
+        expect(amazon_2015_metric_value_card.content).to eq('13')
+        expect(apple_2015_metric_value_card.content).to eq('9')
+        expect(sony_2015_metric_value_card.content).to eq('62')
 
       end
       context "input company doesn't exist in wikirate" do
-        it "should create company and the value" do
-          Card::Env.params[:metric_values] = {"Cambridge"=>["800"]}
-          Card::Env.params[:corrected_company_name] = {"Cambridge"=>"Cambridge University"}
-          source_file = @source.fetch :trait=>:file
-          source_file.update_attributes :subcards=>{"#{@source.name}+#{Card[:metric].name}"=>{:content=>'[[Access to Nutrition Index+Marketing Score]]',:type_id=>Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{:content=>'[[2015]]',:type_id=>Card::PointerID}}
-          expect(Card.exists?("Cambridge University")).to be true
-          expect(Card.exists?("Access to Nutrition Index+Marketing Score+Cambridge University+2015")).to be true
-
-          cambridge_2015_metric_value_card = Card["Access to Nutrition Index+Marketing Score+Cambridge University+2015+value"]
-          expect(cambridge_2015_metric_value_card.content).to eq("800")
+        it 'should create company and the value' do
+          Card::Env.params[:metric_values] = {"Cambridge"=>['800']}
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.